pnr/nextpnr/fpga_interchange/build.sh

```diff
@@ -43,3 +44,5 @@ for device in $DEVICES; do
 done
 
 make install
+
+cp bba/bbasm ${PREFIX}/bin
```

Comment:
Is it me or is something broken with this PR? I can see this line in the script already. This file has 16 lines https://github.com/hdl/conda-eda/blob/master/pnr/nextpnr/fpga_interchange/build.sh

Reply:
Maybe you could rebase it. This PR clearly does not see changes from 7561ee690c4c9f721ca222886921837c0b47b376

Reply:
Oh, right, this PR can actually be closed, as https://github.com/hdl/conda-eda/pull/112 superseded this.